definitions/types/cosmosdb-sql-authentication.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "title": "CosmosDB SQL Authentication",
+  "description": "CosmosDB SQL connection string",
+  "additionalProperties": false,
+  "required": [
+    "username",
+    "password",
+    "hostname",
+    "port"
+  ],
+  "properties": {
+    "username": {
+      "type": "string"
+    },
+    "password": {
+      "type": "string"
+    },
+    "hostname": {
+      "type": "string"
```

Comment:
might be able to use the HTTPS ref here if its guaranteed to be an HTTPS endpoint (I think I saw Cory mention that you query it over HTTPS?

Reply:
Yea I can make that change.